invenio_rdm_records/fixtures/demo.py

```diff
@@ -113,6 +114,12 @@ def fake_language(cls):
         random_id = random.choice(["eng", "aah", "aag"])
         return {"id": random_id}
 
+    @classmethod
+    def fake_creatibutor_role(cls):
+        """Generate random creators_role."""
+        random_id = random.choice(["other", "datacollector", "contactperson"])
```

Comment:
This is could actually speed up a lot the fixtures if we start doind this "hardcoded" instead of reading files... I wonder what would happen if someone overwrites them, then we might be adding some id that doesn't exists.... But then they should not use demo data? To be seen later on.